.travis.yml

```diff
@@ -1,37 +1,62 @@
-# For more information about the configurations used
-# in this file, please see the Travis CI documentation:
 # https://docs.travis-ci.com/
 
-language: node_js
-node_js: node
-sudo: required
+language: minimal
 
-env:
-  global:
-    - secure: "itatOgspOE49LX1fczgut51/3AGiCE7ueMeUHdGseB7cWfyFlJ2eUozspXBfbmWtEls8Y97/V1Q8vZty/WXJ2Q3CdxjK8TBBhVycRgK5Qp03eG/yFy/G5hXN1mGil4KwIwqHVYAQvCVcuDv0Ek4V+PEfNp8KGkRILxP2a6kRjgw="
-
-jobs:
-  include:
-    - env: # Tests for Apache 2.4.x
-       - APACHE_VERSION="2.4"
-       - CONF_TARGET="000-default.conf"
-      before_script: ./bin/setup.sh
-
-    - stage: distribution
-      if: branch = master
-      script: .travis/update-dist.sh
-
-  fast_finish: true
-  allow_failures:
-    - stage: distribution
-
-after_failure:
-  - sudo cat /var/log/apache2/error.log
-  - sudo cat /var/log/apache2/access.log
+services:
+  - docker
 
 addons:
   hosts:
-    - travis.localhost
+    - server.localhost
+    - www-server.localhost
+    - www.server.localhost
+    - secure.server.localhost
+    - www.secure.server.localhost
+  apt:
+    packages:
+      - docker-ce
+
+install:
+  - curl -sSL https://github.com/h5bp/server-configs-test/releases/download/1.0.1/server-configs-test.tar.gz | tar -xz -C test/
+
+before_script:
+  - bash -c "./bin/build.sh test/fixtures/.htaccess test/build/htaccess_fixture.conf"
+  - docker pull httpd
+  - |
+    docker run -d -p 80:80 -p 443:443 --name server \
+      -v $TRAVIS_BUILD_DIR/test/fixtures:/usr/local/apache2/htdocs \
+      -v $TRAVIS_BUILD_DIR/test/vhosts/httpd.conf:/usr/local/apache2/conf/httpd.conf \
+      httpd
+
+#      -v $TRAVIS_BUILD_DIR/test/vhosts:/usr/local/apache2/vhosts \
```

Comment:
Maybe remove the commented out lines here?

Reply:
I will restore them later but I can cleanup for now, sure.